server/src/main/java/org/opensearch/indices/replication/copy/ReplicationCollection.java

```diff
@@ -35,7 +35,7 @@
 import org.apache.logging.log4j.Logger;
```

Comment:
Should we also rename this class to SegmentReplicationCollection?

Reply:
I haven't yet touched the Collection classes so i'd like to leave this as-is for now. In a follow up refactor, we can rename this appropriately after synergizing functionality with the Recovery equivalent